src/NuGet.Clients/NuGet.Console/WpfConsole/WpfConsoleKeyProcessor.cs

```diff
@@ -360,7 +360,7 @@ protected override int InternalExec(ref Guid pguidCmdGroup, uint nCmdID, uint nC
                                 }
                                 else
                                 {
-                                    NuGetUIThreadHelper.JoinableTaskFactory.RunAsync(async delegate { await TriggerCompletionAsync(); });
```

Comment:
I think the safer approach is to not change these and just suppress the warning and get it handled later.